exemplar/src/main/kotlin/com/squareup/exemplar/ExemplarModule.kt

```diff
@@ -1,12 +1,14 @@
 package com.squareup.exemplar
 
 import com.google.inject.AbstractModule
+import com.squareup.exemplar.actions.HelloWebAction
+import com.squareup.exemplar.actions.HelloWebPostAction
+import com.squareup.exemplar.actions.ToggleManualHealthCheckAction
+import com.squareup.exemplar.healthchecks.ManualHealthCheck
+import misk.healthchecks.HealthChecksModule
 import misk.metrics.web.MetricsJsonAction
 import misk.web.WebActionModule
-import misk.web.actions.ReadinessCheckAction
-import misk.web.actions.InternalErrorAction
-import misk.web.actions.LivenessCheckAction
-import misk.web.actions.NotFoundAction
+import misk.web.actions.*
```

Comment:
Please fix intellij settings to avoid star imports.  You can find a settings file at https://github.com/square/java-code-styles